### PR TITLE
Fix linting for issue #2100

### PR DESCRIPTION
**Pull request by MrWako**
**What this PR does / why we need it**:
Fixes a few linting issues

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: 

fixes #2100
fixes [https://github.com/openebs/openebs/issues/2100](https://github.com/openebs/openebs/issues/2100)

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by MrWako** [comment]
this is for [https://github.com/openebs/openebs/issues/2100](https://github.com/openebs/openebs/issues/2100)

File: pkg/version/version.go [hunk 1]
```diff
@@ -40,6 +40,8 @@ var (
 	buildMetaFile = "/src/github.com/openebs/maya/BUILDMETA"
 )
 
+// GetVersion returns the version from the global Version variable,
+// or if this unset, from the VERSION file at the root of the repo
```
Comment (princerachit) [inline pkg/version/version.go]: Rewrite `if Version is unset then from the VERSION file at the root of the repo`

File: pkg/version/version.go [hunk 2]
```diff
@@ -53,6 +55,8 @@ func GetVersion() string {
 	return strings.TrimSpace(string(vBytes))
 }
 
+// GetBuildMeta returns the build type from the global VersionMeta variable,
+// or if this unset, from the BUILDMETA file at the root of the repo
```
Comment (princerachit) [inline pkg/version/version.go]: same as above comment

File: pkg/version/version.go [hunk 3]
```diff
@@ -66,6 +70,8 @@ func GetBuildMeta() string {
 	return "-" + strings.TrimSpace(string(vBytes))
 }
 
+// GetGitCommit returns the Git commit SHA-1 from the global GitCommit variable, 
+// or if this unset, by calling Git directly
```
Comment (princerachit) [inline pkg/version/version.go]: same as previous

File: pkg/version/version.go (hunk 1 above)
Comment (MrWako) [reply pkg/version/version.go]: pushed 2nd commit with rewording - thanks.

File: pkg/version/version.go (hunk 2 above)
Comment (MrWako) [reply pkg/version/version.go]: pushed 2nd commit with rewording - thanks.

File: pkg/version/version.go (hunk 3 above)
Comment (MrWako) [reply pkg/version/version.go]: pushed 2nd commit with rewording - thanks.

**Comment by MrWako** [comment]
@princerachit just pushed a second commit for the requested changes.

**Pull request merged**